pkg/cmd/tknpac/bootstrap/bootstrap.go

```diff
@@ -301,7 +308,9 @@ func addGithubAppFlag(cmd *cobra.Command, opts *bootstrapOpts) {
 	cmd.PersistentFlags().StringVar(&opts.GithubOrganizationName, "github-organization-name", "", "Whether you want to target an organization instead of the current user")
 	cmd.PersistentFlags().StringVar(&opts.GithubApplicationName, "github-application-name", "", "GitHub Application Name")
 	cmd.PersistentFlags().StringVar(&opts.GithubApplicationURL, "github-application-url", "", "GitHub Application URL")
-	cmd.PersistentFlags().StringVarP(&opts.GithubAPIURL, "github-api-url", "", "", "Github Enterprise API URL")
+	cmd.PersistentFlags().StringVarP(&opts.GithubAPIURL, "github-hostname", "", "", "Github Enterprise Hostname ")
+	cmd.PersistentFlags().StringVarP(&opts.GithubAPIURL, "github-api-url", "", "", "Github Enterprise API Host or URL")
```

Comment:
Is this kind of a breaking change for those who relay on `github-api-url` :thinking:

Reply:
that is why i kept it there... but makred as hidden..

Reply:
I misunderstood that sorry

Reply:
Can we add a doc for this ?

Reply:
i think its better to mark deprecated also

Reply:
how would you think i do this?

Reply:
i mean like

```
_ = c.Flags().MarkDeprecated("github-api-url", "please use --github-hostname flag instead")
```

so this will give warning to user to move to new flag

Reply:
ah yes sounds good thanks